src/pipeline/src/etl/transform/transformer/greptime.rs

```diff
@@ -374,13 +374,13 @@ fn json_value_to_row(
 /// 3. The pipeline assumes that the json format is fixed
 /// 4. The pipeline will return an error if the same column datatype is mismatched
 /// 5. The pipeline will analyze the schema of each json record and merge them to get the final schema.
-pub fn identity_pipeline(array: Vec<serde_json::Value>) -> Result<Rows> {
+pub fn identity_pipeline(array: Vec<serde_json::Value>) -> Result<(SchemaInfo, Vec<Row>)> {
```

Comment:
seems like we don't need to modify this. Or we can add a new argument `table: Option<Arc<Table>>` and build the `Rows` inside this method.

Reply:
The reason I didn't pass table is because I didn't want to introduce the table struct in the pipeline crate.

Reply:
Sounds reasonable. Is `Schema` or `SchemaInfo` capable to carry that information?

Reply:
Maybe it would be better to pass a tag field name vec(`schema.row_key_column_names()`) to identity_pipeline.